.travis.yml

```diff
@@ -31,7 +25,10 @@ git:
   submodules: true
 
 before_install:
-  - git submodule update --init
+  - curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.8.13.deb -o elasticsearch.deb
```

Comment:
Is there an apt mirror for elasticsearch v6 that we can use so we don't have to download and install everything manually?

Reply:
Travis needs to whitelist apt mirrors I think. The issue is following doesn't work on travis:
```
apt:
    sources:
      - elasticsearch-6.x
```
https://travis-ci.community/t/upgrade-elasticssearch-version-to-6-x-7-x/3423

We will need to install it manually. what we change here is dependence on artifacts.elastic.co by keeping the deb file in our s3.